doc/developer-guides/hld/virtio-console.rst

```diff
@@ -198,3 +201,31 @@ The File backend only supports console output to a file (no input).
 #. Add the console parameter to the guest OS kernel command line::
 
       console=hvc0
+
+SOCKET
+======
+
+When virtio-console socket_type is appointed to server, device model
+will create a unix domain socket for server VM to bind and listen.
+Then other VM virtio-console socket_type is set to client, these
+VM can connect to the unix domain socket and communicate with server VM.
```

Comment:
The virtio-console socket-type can be set as socket server or client. Device model will create an unix domain socket if appointed the socket_type as server, then server VM or another user VM can bind and listen for communication requirement. If appointed to client, please make sure the socket server is ready prior to launch device model.